publisher/src/main/java/com/pavlovmedia/oss/jaxrs/publisher/impl/swagger/SwaggerConfiguration.java

```diff
@@ -59,23 +51,23 @@
     public String licenseUrl;
     
     @Activate
-    protected void activate(final Map<String,Object> properties) {
+    protected void activate(final SwaggerConfigurationConfig properties) {
         parse(properties);
     }
     
     @Modified
-    protected void modified(final Map<String,Object> properties) {
+    protected void modified(final SwaggerConfigurationConfig properties) {
         parse(properties);
     }
     
-    private void parse(final Map<String,Object> properties) {
-        title = (String) properties.get("title");
-        description = (String) properties.get("description");
-        apiVersion = (String) properties.get("apiVersion");
-        contactName = (String) properties.get("contactName");
-        contactUrl = (String) properties.get("contactUrl");
-        contactEmail = (String) properties.get("contactEmail");
-        licenseName = (String) properties.get("licenseName");
-        licenseUrl = (String) properties.get("licenseUrl");
+    private void parse(final SwaggerConfigurationConfig properties) {
+        title = (String) properties.title();
+        description = (String) properties.description();
+        apiVersion = (String) properties.apiVersion();
+        contactName = (String) properties.contactName();
+        contactUrl = (String) properties.contactUrl();
+        contactEmail = (String) properties.contactEmail();
+        licenseName = (String) properties.licenseName();
+        licenseUrl = (String) properties.licenseUrl();
```

Comment:
Why are we casting these?  The old version needed, but with an OCD you probably don't

Reply:
You're right. I will fix that.